### PR TITLE
fix(LIVE-10131): stop throwing new trc20 account error

### PR DESCRIPTION
**Pull request by cng-ledger**
### ✅ Checklist

- [x] `npx changeset` was attached.
- [ ] **Covered by automatic tests.** 
- [ ] **Impact of the changes:** 
  - ...

### 📝 Description

Stop throwing `TronSendTrc20ToNewAccountForbidden` in Tron bridge for TRC20 support

### ❓ Context

- **JIRA or GitHub link**: [LIVE-10131](https://ledgerhq.atlassian.net/browse/LIVE-10131)

---

### 🧐 Checklist for the PR Reviewers

- **The code aligns with the requirements** described in the linked JIRA or GitHub issue.
- **The PR description clearly documents the changes** made and explains any technical trade-offs or design decisions.
- **There are no undocumented trade-offs**, technical debt, or maintainability issues.
- **The PR has been tested** thoroughly, and any potential edge cases have been considered and handled.
- **Any new dependencies** have been justified and documented.
- **Performance** considerations have been taken into account. (changes have been profiled or benchmarked if necessary)

[LIVE-10131]: https://ledgerhq.atlassian.net/browse/LIVE-10131?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by CremaFR — DISMISSED** [review]
LGTM no opinion on send to TronSendTrc20ToNewAccountForbidden

File: libs/ledger-live-common/src/families/tron/bridge.integration.test.ts [hunk 1]
```diff
@@ -481,27 +480,6 @@ const tron: CurrenciesData<Transaction> = {
...
-          expectedStatus: {
-            amount: new BigNumber("1000000"),
-            errors: {
-              recipient: new TronSendTrc20ToNewAccountForbidden(),
-            },
-            warnings: {},
-            totalSpent: new BigNumber("1000000"),
-            estimatedFees: new BigNumber("0"),
-          },
-        }, // FIXME account have moved...
```
Comment (lambertkevin) [inline libs/ledger-live-common/src/families/tron/bridge.integration.test.ts]: This should still be valid I think, so it's unnecessary to remove it

**Review by lambertkevin — COMMENTED** [review]
Can you please check if letting the test you removed from the bridge integration test is still problematic now that the code removal is only at the signOperation level ?

**Review by CremaFR — APPROVED** [review]

**Review by lambertkevin — APPROVED** [review]
👍

**Pull request merged**